### DISCUSSION TITLE
怎么配置ssl证书访问啊？支持ssl吗？

### DISCUSSION THREAD

**Discussion by zc16607** [nianhua99/PandoraHelper#137] [Q&A] [answered]
helper程序，windows本地运行，可以加证书吗，没看到文档上有？

**Comment by nianhua99** [comment]
windows本地运行，是要在局域网内用ip访问吗

**Reply by zc16607** [reply]
端口映射，外网访问，cf的域名解析过去的

**Reply by nianhua99** [reply]
cf tunnel？那直接用cf的证书可以不，我没试过

**Reply by zc16607** [reply]
如果不支持https的话，那我就只有在中间加一层nginx来做https了。现在是直接cf -> pandoraHelper，加一层就是cf -> nginx -> pandoraHelper

**Reply by zc16607** [reply]
cf本身是有https，但是要你的程序本身也是https才行。

**Reply by zc16607** [reply]
我用的是cf的origin rules，重写端口实现的内网访问

**Comment by zc16607** [comment, marked as answer]
倒腾了一天，现在用了cf tunnel来穿透，效果真不错啊，刚开始用origin rules，接口响应居然达到2-5秒，下载速度倒是能跑满上行，但是响应慢打开网页速度不理想。后面尝试了tunnel，真的是相见恨晚，接口响应差不多在450ms，虽然不能和ipv4直连相比（120ms），但是在能接受的范围了，并且可以不开放端口，安全性提高了很多。
ps: 使用tunnel的话，应用程序本身http就行，cf会自动给网页加ssl证书。
